docs/en/user_guides/inference.md

```diff
@@ -17,8 +17,11 @@ import mmcv
 from mmcv.transforms import Compose
 from mmengine.utils import track_iter_progress
 from mmdet.registry import VISUALIZERS
+from mmdet.utils import register_all_modules
 from mmdet.apis import init_detector, inference_detector
 
+# Register all modules in mmdet into the registries
+register_all_modules()
```

Comment:
replace this with
```
init_default_scope(config.get('default_scope', 'mmdet'))
```
after load the config

Reply:
We no longer need to call register_all_modules after introducing the 'auto_import' in PR https://github.com/open-mmlab/mmdetection/pull/9143.

Reply:
yes